### PR TITLE
Adds submitted and aborted status to Metadata

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala [hunk 1]
```diff
@@ -192,6 +194,7 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
         case None =>
           // All cool, if we got this far the workflow ID was found in the workflow store so this workflow must have never
           // made it to the workflow manager.
+          pushCurrentStateToMetadataService(id, WorkflowAborted)
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: Presumably the `Aborted` is already sent to the metadata for workflows that had already started?

**Comment by cjllanwarne** [comment]
👍

File: engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala (hunk 1 above)
Comment (Horneth) [inline engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: Yes, this is for workflows that have not been started yet and are sitting in the workflow store.

**Comment by ruchim** [comment]
:+1:

**Comment by Horneth** [comment]
@helgridly This will fix your missing status in metadata issue

**Comment by helgridly** [comment]
Awesome! This makes me very happy.

**Pull request merged**